lambda/writePark/index.js

```diff
@@ -1,30 +1,30 @@
 const AWS = require('aws-sdk');
-
+ 
```

Comment:
Nit: remove un-needed spaces. (We use prettier on default settings for our formatter for JS)

Reply:
Setup prettier to autoformat and remove the spaces. Should be good now.